app/models/user.rb

```diff
@@ -16,4 +18,9 @@ def normalize_queued_videos_order
   def video_queued?(video)
     video ? (queued_videos & video.queued_videos).any? : false
   end
+
+  def has_follower?(other_user)
+    return false unless other_user
```

Comment:
Oops, when I saw your test for this I went exploring. I had forgotten that `nil` counts as an argument. Sorry about that :smiley: